Buffer size fixes

leechuck opened the pull request:
The patches increase buffer size throughout func; buffer sizes may be sufficient for use in GO, but will not work with ontologies using longer IDs or full URIs (currently ID size limited to 20 chars), and in ontologies with long labels (>200); increased both ID length and label length to 20000 chars.

wjv commented:
Hey @leechuck! 😀 Thanks to you, @kaypruefer has finally joined GitHub! I’ll leave it to him to address the issue…

leechuck commented:
Hi @wjv and welcome @kaypruefer. I just updated the request a bit more (forgot to update the buffer size in its declaration for wilcoxon and binomial; also updated the 2x2 contingency test). Let me know if I can be of any more help.

kaypruefer commented:
Fixed in latest version
